common/src/test/java/tachyon/worker/block/io/LocalFileBlockWriterTest.java

```diff
@@ -29,24 +29,43 @@
 
 import tachyon.util.io.BufferUtils;
 
+/**
+ * Tests for the {@link LocalFileBlockWriter} class.
+ */
 public class LocalFileBlockWriterTest {
   private static final long TEST_BLOCK_SIZE = 1024;
 
   private LocalFileBlockWriter mWriter;
   private String mTestFilePath;
 
+  /**
+   * Rule for the temporary folder.
```

Comment:
This can go on one line and be more descriptive about what the TemporaryFolder rule does, see previous comment